appserver/pom.xml

```diff
@@ -115,8 +114,7 @@
     <modules>
         <module>common</module>
         <module>libpam4j</module>
-	    <module>jaxr-ra</module>
-	    <module>ldapbp</module>
+	<module>ldapbp</module>
```

Comment:
The extra add lines appear to be due to the indentation being filled in with tabs rather than spaces.